studio/lib/helpers.ts

```diff
@@ -165,33 +165,57 @@ export const copyToClipboard = (str: string, callback = () => {}) => {
   }
 }
 
-export async function passwordStrength(value: string) {
+type PasswordStrengthResponse = {
+  message: string
+  warning: string | null
+  strength: number
+}
+export async function passwordStrength(value: string): Promise<PasswordStrengthResponse> {
   let message = ''
   let warning = ''
   let strength = 0
 
-  if (value && value !== '') {
-    const response = await post(`${API_URL}/profile/password-check`, { password: value })
-    if (!response.error) {
-      const { result } = response
-      const score = (PASSWORD_STRENGTH as any)[result.score]
-      const suggestions = result.feedback?.suggestions ? result.feedback.suggestions.join(' ') : ''
-
-      // set message :string
-      message = `${score} ${suggestions}`
+  if (!value) {
+    return {
+      message,
+      warning,
+      strength,
+    }
+  }
```

Comment:
If no value, return an empty message/warning?

Reply:
That's what the existing behavior was, so I just kept it.  Returning null or something more explicit might be better, but I was trying to keep this just focused on the max length change.